.github/workflows/ci.yml

```diff
@@ -129,8 +136,10 @@ jobs:
       - name: Install python dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install tox tox-gh-actions coverage[toml]
+          python -m pip install hatch
       - name: Run python tests
-        run: tox
+        run: hatch run +mistune +python=${{ matrix.python }} test:test-python
```

Comment:
It seems `tox.ini` isn't needed anymore.

Reply:
The reason I hesitate to get rid of tox is that hatch currently doesn't have any equivalent of `tox -p` — that is, it doesn't yet have the ability to run the various test environments in parallel (though, reportedly, that feature is planned). For the CI tests, it doesn't matter, but it does matter for the dev running the tests on their workstation.
And if we're going to keep the tox tests for that, I figure we should use them in CI.